internal/venice-common/src/main/java/com/linkedin/venice/writer/LeaderCompleteState.java

```diff
@@ -0,0 +1,28 @@
+package com.linkedin.venice.writer;
+
+/**
+ * Completion state of the leader partition.
+ */
+public enum LeaderCompleteState {
+  /**
+   * Default state for the standby to know that the feature is not supported and
+   * don't consider leader's completion status for its completion.
+   */
+  NOT_SUPPORTED,
```

Comment:
It will be used as default in pcs for the standby replica to be able to differentiate between whether this header is seen or not (rather than having a separate flag). It will be used in 2nd diff.